Pod/Classes/Views/Gradient/GradientView.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  GradientView.swift
+//  Swiftilities
+//
+//  Created by Nick Bonatsakis on 5/1/17.
+//  Copyright © 2017 Raizlabs. All rights reserved.
+//
+
+import UIKit
+
+/// A basic view that wraps CAGradientLayer
+public class GradientView: UIView {
+
+    /// The direction of the gradient
+    ///
+    /// - horizontal: Horizontal direction.
+    /// - vertical: Vertical direction.
+    /// - custom: Provide start and end CGPoint to specify a custom direction.
+    public enum Direction {
+        case horizontal
+        case vertical
+        case custom(start: CGPoint, end: CGPoint)
+
+        var start: CGPoint {
+            switch self {
+            case .horizontal: return CGPoint(x: 0, y: 0.5)
```

Comment:
Confused, 0.5 signifies middle. If you used only 0 and 1 you'd get corners only (e.g. diagonal gradients).